rakan/cRakan/src/Runner.cc

```diff
@@ -62,8 +62,8 @@ void Runner::add_node(uint32_t node_id,
 }
 
 bool Runner::add_edge(uint32_t node_one, uint32_t node_two) {
-  if (graph_->ContainsNode(graph_->GetNode(node_one)) && 
-      graph_->ContainsNode(graph_->GetNode(node_two))) {
+  if (graph_->ContainsNode(graph_->GetNode(node_one)->id_) && 
+      graph_->ContainsNode(graph_->GetNode(node_two)->id_)) {
```

Comment:
Can simplify this with just `graph_->ContainsNode(node_one)` and the same for `node_two`

Reply:
oh lmao